core/chains/evm/txmgr/models.go

```diff
@@ -16,21 +16,25 @@ import (
 
 	"github.com/smartcontractkit/chainlink/core/assets"
 	"github.com/smartcontractkit/chainlink/core/chains/evm/gas"
+	"github.com/smartcontractkit/chainlink/core/logger"
 	cnull "github.com/smartcontractkit/chainlink/core/null"
 	"github.com/smartcontractkit/chainlink/core/services/pg/datatypes"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
+// EthTxMeta contains fields of the transaction metadata
 type EthTxMeta struct {
-	JobID         int32
-	RequestID     common.Hash
-	RequestTxHash common.Hash
+	JobID         int32       `json:"JobID"`
+	RequestID     common.Hash `json:"RequestID"`
+	RequestTxHash common.Hash `json:"RequestTxHash"`
 	// Used for the VRFv2 - max link this tx will bill
 	// should it get bumped
-	MaxLink string
+	MaxLink *string `json:"MaxLink,omitempty"`
```

Comment:
Why change this type to be a pointer to string?

Reply:
This is the VRF specific field. Going to be added to the logger only if a TX is related to VRF.